src/Files.App/Services/Windows/WindowsJumpListService.cs

```diff
@@ -46,7 +46,10 @@ public async Task AddFolderAsync(string path)
 						await instance.SaveAsync();
 					}
 				}
-				catch { }
+				catch (Exception ex)
+				{
+					App.Logger.LogWarning(ex, ex.Message);
+				}
```

Comment:
This is simply adding log output, which doesn't help avoid crashes.
I think we need to expand the try block.

Reply:
I'll do that, but it will be nice to fix these errors.

There are some reports about jumplists not populating.